strawberry/relay/types.py

```diff
@@ -467,10 +478,8 @@ def resolve_id(
         You can override this method to provide a custom implementation.
 
         Args:
-            info:
-                The strawberry execution info resolve the type name from
-            root:
-                The node to resolve
+            info: The strawberry execution info resolve the type name from
+            root: The node to resolve
```

Comment:
nit: the other docstrings use periods at the end

```suggestion
            info: The strawberry execution info resolve the type name from.
            root: The node to resolve.
```